### ISSUE TITLE
Importing Plugins in Flow

### ISSUE THREAD

**Issue by hexlo** [HaveAGitGat/Tdarr#1050]
Please put plugin requests/bugs at: https://github.com/HaveAGitGat/Tdarr_Plugins

**Is your feature request related to a problem? Please describe.**
I'm trying to create a Flow for my media library. I'd like to use the classic plugins used in the Classic Plugin Stack in my Flow. The only way to import them is to create a new Flow and import a stack as a new template.

**Describe the solution you'd like**
I'd like to be able to import plugins (classic or not) in any Flow.

**Describe alternatives you've considered**
The import "Classic Plugin Stack" option could be available in the Flow editor, instead of only being available when you create a new Flow.

**Additional context**
![Screenshot 2024-07-23 182000](https://github.com/user-attachments/assets/b6189b5e-a8bf-4d89-a331-441d8068c003)

**Comment by supersnellehenk** [comment]
If you're meaning to run a classic plugin in your flow, use the 'run classic transcode plugin' block. https://github.com/HaveAGitGat/Tdarr_Plugins/blob/71f468fab7ca51613dfee13c42e09435cbef7b8a/FlowPluginsTs/CommunityFlowPlugins/classic/runClassicTranscodePlugin/2.0.0/index.ts

Otherwise I'm not really sure what you mean, as importing should be a one time thing from the plugin stack to flows.

**Comment by HaveAGitGat** [comment]
Yes what @supersnellehenk is correct. Not sure what you mean by `classic or not`. Flow plugins can be seen on the Flows tab sidebar and to run a classic plugin in a flow use `Run Classic Transcode Plugin` or `Run Classic Filter Plugin`:

![image](https://github.com/user-attachments/assets/af75e89a-cf33-4fa2-a6a2-2f989c8ab042)
